### PR TITLE
fix: form view field validation error message, use alias name instead of original field name

### PR DESCRIPTION
**Pull request by gitstart**
## Change Summary
fixes: #4919 

use alias name in error message, not original field name

## Change type

- [x] fix: (bug fix for the user, not a fix to a build script)

## Additional information / screenshots (optional)
Screenshot 2023-01-26 at 11 36 29

Anything for maintainers to be made aware of

### CODE REVIEW

**Review by wingkwong — COMMENTED** [review]
1. Next time please leave a comment in the issue first to ask for assignment to avoid duplicate work (as it has been assigned to someone already).
2. Your change is only partially correct. If I don't set a label, submitting without data would show `null is required`. 
3. The same change should apply to virtual columns as well. You can test it by creating a LTAR column.

**Comment by gitstart** [comment]
> 1. Next time please leave a comment in the issue first to ask for assignment to avoid duplicate work (as it has been assigned to someone already).
> 2. Your change is only partially correct. If I don't set a label, submitting without data would show `null is required`.
> 3. The same change should apply to virtual columns as well. You can test it by creating a LTAR column.

This is noted

**Comment by gitstart** [comment]
Hi @wingkwong, the changes has been updated

File: packages/nc-gui/components/smartsheet/Form.vue [hunk 1]
```diff
@@ -662,7 +662,12 @@ watch(view, (nextView) => {
                     v-if="isVirtualCol(element)"
                     :name="element.title"
                     class="!mb-0"
-                    :rules="[{ required: isRequired(element, element.required), message: `${element.title} is required` }]"
+                    :rules="[
+                      {
+                        required: isRequired(element, element.required),
+                        message: `${element.label ? element.label : element.title} is required`,
```
Comment (wingkwong) [inline packages/nc-gui/components/smartsheet/Form.vue:668]: `${element.label || element.title}`

File: packages/nc-gui/components/smartsheet/Form.vue [hunk 2]
```diff
@@ -679,7 +684,12 @@ watch(view, (nextView) => {
                     v-else
                     :name="element.title"
                     class="!mb-0"
-                    :rules="[{ required: isRequired(element, element.required), message: `${element.title} is required` }]"
+                    :rules="[
+                      {
+                        required: isRequired(element, element.required),
+                        message: `${element.label ? element.label : element.title} is required`,
```
Comment (wingkwong) [inline packages/nc-gui/components/smartsheet/Form.vue:690]: `${element.label || element.title}`

**Review by wingkwong — CHANGES_REQUESTED** [review]

File: packages/nc-gui/components/smartsheet/Form.vue (hunk 1 above)
Comment (gitstart) [reply packages/nc-gui/components/smartsheet/Form.vue]: it has been updated

File: packages/nc-gui/components/smartsheet/Form.vue (hunk 2 above)
Comment (gitstart) [reply packages/nc-gui/components/smartsheet/Form.vue]: it has been updated

**Review by wingkwong — APPROVED** [review]

**Pull request merged**